drivers/classic/src/mtclim_wrapper.c

```diff
@@ -233,7 +240,7 @@ mtclim_init(int               have_dewpt,
             log_err("have_dewpt not yet implemented ...");
         }
     }
-    tinystepspday = CONST_CDAY / param.MTCLIM_SRADDT;
+    tinystepspday = (size_t)(CONST_CDAY / param.MTCLIM_SRADDT);
```

Comment:
Check that this division is OK as is (referring to earlier discussion of `#define`d values)